src/pages/RegistrationFlow/LogoRF/LogoRF.tsx

```diff
@@ -0,0 +1,11 @@
+import "./LogoRF.scss";
+
+export const LogoRF = () => (
```

Comment:
and why is this logo is in the registration flow? what way is that related to the external link?

Reply:
wrong location, moved it to organisms fc594aa